_posts/2023-11-03-ccs-wallet-incident.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Community Crowdfunding System wallet incident
+summary: 2,675.73 XMR (the entire balance) was drained from the CCS wallet on September 1, 2023. The General Fund wallet remains intact.
+tags: [urgent, announcements, core]
+author: Monero Core Team
+---
+
+During a routine transfer, we noticed the '[Community Crowdfunding System](https://ccs.getmonero.org/)' (CCS) main Wallet was drained of 2,675.73 XMR (the entire balance) on September 1, 2023. The hot wallet, used for payments to contributors, is untouched; its balance is ~244 XMR. We have thus far not been able to ascertain the source of the breach. An internal investigation is ongoing and further details will be released once we have them.
```

Comment:
```suggestion
During a routine transfer, we noticed the '[Community Crowdfunding System](https://ccs.getmonero.org/)' (CCS) Donation Wallet was drained of 2,675.73 XMR (the entire balance) on September 1, 2023. A second wallet, used for payments to proposers, is untouched; its balance is ~244 XMR. We have thus far not been able to ascertain the source of the breach. An internal investigation is ongoing and further details will be released once we have them.
```

Reply:
main Wallet mixed capitalization
clarification that the hot wallet is not the same wallet mentioned previously

Reply:
What about using cold and hot wallet terms?

Reply:
They're both hot, technically.

Reply:
proposers become contributors, then they receive moneys. both wallets are hot. there was a main + secondary wallet. 

replacing "hot" with "secondary" + fixing mixed capitalisation should be OK